Functional/billingSystem_Func.py

```diff
@@ -0,0 +1,94 @@
+from functools import reduce
+
+def get_room_types():
+    return {
+        1: {"type": "Single", "rate": 50},
+        2: {"type": "Double", "rate": 90},
+        3: {"type": "Suite", "rate": 150},
+    }
```

Comment:
please use mysql database instead if JSON
you can see dummy data file

Reply:
ok